content/applications/finance/expenses/expense_reports.rst

```diff
@@ -0,0 +1,109 @@
+===============
+Expense reports
+===============
+
+When expenses are ready to submit (such as at the end of a business trip, or once a month), an
+*expense report* needs to be created. Navigate to the main :menuselection:`Expenses` app dashboard,
+which displays a default :guilabel:`My Expenses` view, or go to :menuselection:`Expenses app --> My
+Expenses --> My Expenses`.
+
+Expenses are color coded by status. Any expense with a status of :guilabel:`To Report` (expenses
+that still need to be added to an expense report) the text appears in blue. All other statuses
+(:guilabel:`To Submit`, :guilabel:`Submitted`, and :guilabel:`Approved`) the text appears in black.
```

Comment:
I meant the physical text, not the color of the tags. Both tags are blue (bad idea, IMO, they should be different) but the actual text of the entry is either blue or black. So the tags that say "To Report" appear in blue, but the ones that are tagged "To Submit" (meaning they are on a report already) appear in black.

Reply:
I am checking with the experts since the image above is definitely off. I know I had some issues as well, when I deleted reports, and tried to force the DB to show me the data I wanted. When I checked on a new runbot, that wasn't messed with, I got it to work exactly as I described. SO I am leaving it as-is, while I wait for the experts to weigh in and let me know if this is jus not working right, or if there's something else I don't know.